tests/kernel/common/src/printk.c

```diff
@@ -97,7 +99,9 @@ void printk_test(void)
 			  "%u %2u %4u %8u\n", 42, 42, 42, 42);
 	count += snprintk(ram_console + count, sizeof(ram_console) - count,
 			  "%u %02u %04u %08u\n", 42, 42, 42, 42);
-
+	count += snprintk(ram_console + count, sizeof(ram_console) - count,
+			  "%-8u%-6d%-4x%-2p%8d\n",
```

Comment:
the %-2p doesn't seem to work as intended as it's still printing: 0x00000042, was it supposed to print "0x42" instead?